python/src/QuantileMatchingFactory_doc.i.in

```diff
@@ -0,0 +1,227 @@
+
+Parameters
+----------
+solver : :class:`~openturns.OptimizationAlgorithm`
+    The solver used for numerical optimization of the moments."
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::QuantileMatchingFactory::getOptimizationAlgorithm
+"Accessor to the solver.
+
+Returns
+-------
+solver : :class:`~openturns.OptimizationAlgorithm`
+    The solver used for numerical optimization of the moments."
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::QuantileMatchingFactory::setOptimizationBounds
+"Accessor to the optimization bounds.
+
+Parameters
+----------
+bounds : :class:`~openturns.Interval`
+    The bounds used for numerical optimization of the likelihood."
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::QuantileMatchingFactory::getOptimizationBounds
+"Accessor to the optimization bounds.
+
+Returns
+-------
+bounds : :class:`~openturns.Interval`
+    The bounds used for numerical optimization of the likelihood."
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::QuantileMatchingFactory::setKnownParameter
+"Accessor to the known parameters.
+
+Parameters
+----------
+values : sequence of float
+    Values of fixed parameters.
+indices : sequence of int
+    Indices of fixed parameters.
+
+Examples
+--------
+>>> import openturns as ot
+>>> ot.RandomGenerator.SetSeed(0)
+>>> distribution = ot.Beta(2.3, 2.2, -1.0, 1.0)
+>>> sample = distribution.getSample(10)
+>>> factory = ot.QuantileMatchingFactory(ot.Beta())
+>>> # set (a,b) out of (r, t, a, b)
+>>> factory.setKnownParameter([-1.0, 1.0], [2, 3])
+>>> inf_distribution = factory.build(sample)"
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::QuantileMatchingFactory::getKnownParameterValues
+"Accessor to the known parameters indices.
+
+Returns
+-------
+values : :class:`~openturns.Point`
+    Values of fixed parameters."
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::QuantileMatchingFactory::getKnownParameterIndices
+"Accessor to the known parameters indices.
+
+Returns
+-------
+indices : :class:`~openturns.Indices`
+    Indices of fixed parameters."
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::QuantileMatchingFactory::buildFromQuantiles
+"Build from quantiles.
+
+Parameters
+----------
+quantiles : sequence of float
+    Quantiles of the distribution, matching the probabilities provided to the constructor.
+
+Returns
+-------
+dist : :class:`~openturns.Distribution`
+    Estimated distribution.
+
+
+Examples
+--------
+>>> import openturns as ot
+>>> probabilities = [0.05, 0.25, 0.75, 0.95]
+>>> distribution = ot.Beta(2.3, 2.2, -1.0, 1.0)
+>>> probabilities = [0.05, 0.25, 0.75, 0.95]
```

Comment:
This produces:
![image](https://user-images.githubusercontent.com/31351465/190730229-1eefa16b-5655-4aa7-8cd5-74772acbc56c.png)

This is why I suggest to remove the second one:
```suggestion
```